### PR TITLE
Fix github issue cleanup

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: cmd/publishing-bot/github.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func ReportOnIssue(e error, logs, token, org, repo string, issue int) error {
 	}
 
 	// create new newComment
-	body := transfromLogToGithubFormat(logs, 50, fmt.Sprintf("The last publishing run failed: %v", e))
+	body := transfromLogToGithubFormat(logs, 50, fmt.Sprintf("/reopen\n\nThe last publishing run failed: %v", e))
```
Comment (nikhita) [inline cmd/publishing-bot/github.go]: Who closes the issue? Bot or us?

Comment (sttts) [reply cmd/publishing-bot/github.go]: Bot

File: cmd/publishing-bot/github.go [hunk 2]
```diff
@@ -65,28 +66,32 @@ func ReportOnIssue(e error, logs, token, org, repo string, issue int) error {
...
+		if *c.ID == *newComment.ID {
+			continue
+		}
+
+		glog.Infof("Deleting comment %d", *c.ID)
+		resp, err = client.Issues.DeleteComment(ctx, org, repo, *c.ID)
+		if err != nil {
+			return fmt.Errorf("failed to delete github comment %d of issue #%d: %v", *c.ID, issue, err)
+		}
+		if resp.StatusCode >= 300 {
```
Comment (yue9944882) [inline cmd/publishing-bot/github.go]: shouldn't we follow the redirection on 3xx? iirc, it's the default behavior of golang http client.

File: cmd/publishing-bot/main.go [hunk 3]
```diff
@@ -56,7 +56,7 @@ func main() {
 	repoName := flag.String("source-repo", "", "the name of the source repository (eg. kubernetes)")
 	repoOrg := flag.String("source-org", "", "the name of the source repository organization, (eg. kubernetes)")
 	targetOrg := flag.String("target-org", "", `the target organization to publish into (e.g. "k8s-publishing-bot")`)
-	basePublishScriptPath := flag.String("base-publish-script-path", "./artifacts/scripts", `the base path in source repo where bot will look for publishing scripts`)
+	basePublishScriptPath := flag.String("base-publish-script-path", "./publish_scripts", `the base path in source repo where bot will look for publishing scripts`)
```
Comment (yue9944882) [inline cmd/publishing-bot/main.go]: 💯

File: cmd/publishing-bot/github.go (hunk 2 above)
Comment (sttts) [reply cmd/publishing-bot/github.go]: yes, in a follow-up this would be fine.

**Comment by sttts** [comment]
With this the bot works again.

File: cmd/publishing-bot/main.go (hunk 3 above)
Comment (zhouhaibing089) [reply cmd/publishing-bot/main.go]: Oh, I updated this before since I can run `publishing-bot` locally under the project root. Sorry for this.

**Pull request merged**